code/game/objects/structures/barricade.dm

```diff
@@ -1137,3 +1142,38 @@
 	var/image/new_overlay = image(icon, src, "[icon_state]_overlay", dir == SOUTH ? BELOW_OBJ_LAYER : ABOVE_MOB_LAYER, dir)
 	new_overlay.pixel_y = (dir == SOUTH ? -32 : 32)
 	. += new_overlay
+
+/*----------------------*/
+// NONFOLDING PLASTEEL
+/*----------------------*/
+
+/obj/structure/barricade/metal/plasteel
+	name = "plasteel barricade"
```

Comment:
can you rename it to be /folding
and /solid
or something because
/plasteel/metal
/metal/plasteel
is very unclear